.dev/scripts/generate_changelog.py

```diff
@@ -0,0 +1,235 @@
+}
+
+
+MODULES = {
+    "cmake": "CMake",
+    "2d": "libpcl_2d",
+    "common": "libpcl_common",
+    "cuda": "libpcl_cuda",
+    "features": "libpcl_features",
+    "filters": "libpcl_filters",
+    "geometry": "libpcl_geometry",
+    "gpu": "libpcl_gpu",
+    "io": "libpcl_io",
+    "kdtree": "libpcl_kdtree",
+    "keypoints": "libpcl_keypoints",
+    "ml": "libpcl_ml",
+    "octree": "libpcl_octree",
+    "outofcore": "libpcl_outofcore",
+    "people": "libpcl_people",
+    "recognition": "libpcl_recognition",
+    "registration": "libpcl_registration",
+    "sample_consensus": "libpcl_sample_consensus",
+    "search": "libpcl_search",
+    "segmentation": "libpcl_segmentation",
+    "simulation": "libpcl_simulation",
+    "stereo": "libpcl_stereo",
+    "surface": "libpcl_surface",
+    "visualization": "libpcl_visualization",
+    "apps": "PCL Apps",
+    "docs": "PCL Docs",
+    "tutorials": "PCL Tutorials",
+    "examples": "PCL Examples",
+    "tests": "PCL Tests",
+    "tools": "PCL Tools",
+    "ci": "CI",
+    None: "Uncategorized",
+}
+
+
+def fetch_latest_release_date():
+    """
+    Fetch the date when the latest release was created.
+    """
+    url = "https://api.github.com/repos/PointCloudLibrary/pcl/releases/latest"
+    response = requests.get(url)
+    data = response.json()
+    if response.status_code != 200:
+        sys.exit(
+            "Failed to fetch release info. Server response: '{}'".format(
+                data["message"]
+            )
+        )
+    return data["created_at"]
+
+
+def fetch_pr_since(start_date):
+    """
+    Fetch data of PRs merged after a given start date.
+    """
+    url = f"https://api.github.com/search/issues?q=is:pr+repo:PointCloudLibrary/pcl+merged:>={start_date}"
+    pr_data = list()
+    page = 1
+    total_count = 1  # for the first iteration assume that total is 1
+    while len(pr_data) < total_count:
+        response = requests.get(f"{url}&page={page}&per_page=100")
+        data = response.json()
+        if response.status_code != 200:
+            sys.exit(
+                "Failed to fetch PR info. Server response: '{}'".format(data["message"])
+            )
+        total_count = data["total_count"]
+        pr_data.extend(data["items"])
+        page += 1
+    return pr_data
+
+
+def filter_labels(labels, prefix):
+    """
+    Filter a given list of PR labels, keeping only those starting with a given prefix.
+    The prefix is stripped from the labels.
+    """
+    return [
+        label["name"][len(prefix) :]
+        for label in labels
+        if label["name"].startswith(prefix)
+    ]
+
+
+def make_pr_bullet_point(pr, prefix=None):
+    ref = "[#{0}](https://github.com/PointCloudLibrary/pcl/pull/{0})".format(
+        pr["number"]
+    )
+    if prefix in ("modules", "categories"):
+        p = pr[prefix]
+    else:
+        p = []
+    prefix = "".join(["[" + k + "]" for k in p])
+    if prefix:
+        prefix = "**" + prefix + "** "
```

Comment:
It can be
```suggestion
    if prefix in ("modules", "categories"):
        prefix = "".join(["[" + k + "]" for k in pr[prefix]])
        prefix = "**" + prefix + "** "
```